compiler-core/src/parse.rs

```diff
@@ -2509,6 +2509,14 @@ where
         }
     }
 
+    fn expect_expression_unit(&mut self) -> Result<UntypedExpr, ParseError> {
+        if let Some(e) = self.parse_expression_unit()? {
+            Ok(e)
+        } else {
+            self.next_tok_unexpected(vec!["An expression unit".into()])
```

Comment:
```suggestion
            self.next_tok_unexpected(vec!["An expression".into()])
```

"expression unit" is a parser implementation detail, it doesn't mean anything to a Gleam user.

Reply:
Gotcha, was wondering about this. Will change it to just expression.